lepton/logger_test.go

```diff
@@ -0,0 +1,152 @@
+package lepton_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/nanovms/ops/lepton"
+)
+
+const (
+	colorRed    = "\033[31m"
```

Comment:
why are these repeated from lepton/logger?

Reply:
didn't make sense to me making these constants values readable from external packages.